sile.1.in

```diff
@@ -0,0 +1,63 @@
+.TH sile 1 "18 October 2019" "version @VERSION@"
```

Comment:
I'd be quite okay with writing this in Markdown and generating the ~~gross~~troff format with Pandoc. Note that Pandoc would not be required to build from the release source tarballs, but it would be required to build the man page from git or to *cut the release* tarballs.
Either way there are pros and cons. We could implement that later too after we see how bad the churn ends up being and hence how painful editing this is.